DUI3/Speckle.Connectors.DUI/Bridge/BrowserBridge.cs

```diff
@@ -296,11 +290,11 @@ private void NotifyUIMethodCallResultReady(string requestId, string? serializedD
   /// <returns></returns>
   public string? GetCallResult(string requestId)
   {
-    bool isFound = _resultsStore.TryRemove(requestId, out string? res);
-    if (!isFound)
-    {
-      throw new ArgumentException($"No result for the given request id was found: {requestId}", nameof(requestId));
-    }
+    _ = _resultsStore.TryRemove(requestId, out string? res);
```

Comment:
I don't remember why we removed `isFound` logic, I would take it back if there is a no reason

Reply:
Good catch, This was not intended to be left in this PR (we removed it briefly for experimental testing of unknown bindings)
I'll make sure it gets back in